src/host/main.cpp

```diff
@@ -235,6 +235,14 @@ int main(int argc, char** argv)
       true)
     ->check(CLI::ExistingFile);
 
+  size_t join_timer = 1000;
+  join->add_option(
+    "--join-timer",
+    join_timer,
+    "Frequency at which the joining node will try to join an existing network "
```

Comment:
Minor: This is a delay (ms), not a frequency. Would prefer something like "Time after which the joining node will resend join requests to existing network (ms)".